docs/playbook/training-modules/Module-01.md

```diff
@@ -0,0 +1,90 @@
+
+ii. Your department name will be `sandbox` for the purpose of the training e.g. `module.department_sandbox`.
+
+iii. Suffix your new Terraform module name with the department `sandbox` and `your_name` so that each participant’s job will be easily identifiable. The module name will need to also reflect the individual dataset so make sure that the dataset name is also included within the module name e.g. `sandbox_joe_bloggs_covid_vaccinations`.
+
+iv. Prefix the dataset_name with your name e.g `joe_bloggs_covid_vaccinations`.
+
+v. No schedule is required as this is a one-off import, so this line does not need to be included within the module.
+
+
+### 4. Getting your jobs deployed to the Data Platform
+The next step is to commit your changes to the [Data Platform GitHub repository](https://github.com/LBHackney-IT/Data-Platform). Follow the instructions [here](https://playbook.hackney.gov.uk/Data-Platform-Playbook/playbook/getting-set-up/using-github#committing-your-changes-to-the-data-platform-project) on how to do this.
+
+> :warning: Please note that if you do not have access to the AWS sandbox account then this step will fail.
+
+Create a [Pull Request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) (PR) for the changes to the Terraform script that you have just made. **Two approvals** are required before your code can get merged into the *main* branch of the Data Platform repository. If you know the names of your reviewers, you can send them the URL of your PR to speed up the process. 
+
+Once you have received confirmation that two reviewers have approved your PR, go back to GitHub and merge your code. This will deploy your job out to Data Platform Pre-Production environment. This only happens once. From that point onwards, you will be able to see it in the AWS console, and it will run on schedule or manually, depending on how it was configured in Terraform. 
+
+>:warning: There will be a 10-15 minute delay whilst the code deployment takes place. You can take a break or look at a ready-made job in the interim. To check the status of your deployment, navigate to the Actions tab in the Github web interface.
+
+In everyday use you may not need the data to be immediately available, in which case steps 5-7 would be picked up by the scheduled daily jobs, provided a schedule is specified in the Terraform module. 
+
+Log in to AWS as the `DataPlatformSandboxStg` role via the `Management Console` for that role.
+
+![AWS console](../images/sandox-console.png)
+### 5. Finding and running your jobs in the AWS console
+
+>:bulb: You can search for AWS tools like AWS Glue, Crawlers etc. using the toolbar. If you cannot find a job or crawler check the region is London in the top right of the screen.
+
+
+Your jobs should now be available within Glue Studio. They will be named like [Google Sheets Import Job - department dataset name].
```

Comment:
Can we link to glue studio job page here

Reply:
The name should be "stg Google Sheet..."

Reply:
Updated